database/java/2017/9793.yaml

```diff
@@ -0,0 +1,17 @@
+cve: 2017-9793
+title: "DoS attack via crafted XML payload processed by REST Plugin using XStream library"
+description: >
+    The REST Plugin is using outdated XStream library which is vulnerable and allow perform a DoS attack using malicious request with specially crafted XML payload.
+cvssv3: 5.9/CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H
```

Comment:
This is inconsistent to `cvss_v2` usage; this needs to be added to the validation script.